api/src/test/java/org/openmrs/module/sync2/api/utils/SyncHashcodeUtilsUtilsTest.java

```diff
@@ -87,26 +92,35 @@ public void hashCodesEqualWhenVoided() throws IOException {
 	public void hashCodesEqualWhenRetired() throws IOException {
 		Provider provider = createProvider();
 		// get a hashcode before the change
-		UUID hashcode = SyncHashcodeUtils.getHashcode(getSimpleObject(provider), Provider.class);
+		String hashcode = SyncHashcodeUtils.getHashcode(getSimpleObject(provider));
 
 		provider.setDateRetired(new Date(1540000000000L));
 		provider.setRetired(true);
 		provider.setRetiredBy(new User());
 		provider.setRetireReason("Retired Reason");
-		UUID hashcode2 = SyncHashcodeUtils.getHashcode(getSimpleObject(provider), Provider.class);
+		String hashcode2 = SyncHashcodeUtils.getHashcode(getSimpleObject(provider));
 
 		Assert.assertNotNull(hashcode);
 		Assert.assertNotNull(hashcode2);
 		Assert.assertEquals(hashcode, hashcode2);
 	}
 
 	@Test
-	public void visitHashCodesEqualsWhenDatabaseIdChanged() throws IOException {
```

Comment:
Please readd this test

Reply:
I changed the approach and removed this test because it is deprecated.